### DISCUSSION TITLE
Fluent-bit launch daemon support

### DISCUSSION THREAD

**Discussion by dspruell** [Homebrew/discussions#3608] [Everyday usage]
Hi! Noticing that the fluent-bit formula seems to install without a means of controlling a running daemon. 

```
$ brew install fluent-bit
==> Downloading https://ghcr.io/v2/homebrew/core/fluent-bit/manifests/1.9.7
==> Downloading https://ghcr.io/v2/homebrew/core/fluent-bit/blobs/sha256:1183bce
==> Pouring fluent-bit--1.9.7.big_sur.bottle.tar.gz
🍺  /usr/local/Cellar/fluent-bit/1.9.7: 150 files, 8.4MB

$ brew services start fluent-bit
Error: Formula `fluent-bit` has not implemented #plist, #service or installed a locatable service file
```

Given that fluent-bit would commonly be run as a background daemon on a system, is it typical that this would be omitted? Wondering if this should be submitted as an issue.

**Comment by carlocab** [comment]
No, probably not typical. However, I'd ask that you open a pull request instead of an issue. In my experience, issues for things like this just go stale and get closed.
